### PR TITLE
added blog

### PR DESCRIPTION
**Pull request by lajbel**
* added in nav blog page
* BlogEntry component
* added images to use as banners
* Added gray-matter package to read the YAML of markdown, separe and don't render that part in the marked page

### CODE REVIEW

**Comment by lajbel** [comment]
apparently i had to remove that "cm" to be able to use fmt because it told me it couldn't find any folder or file named like that. just in case it is a windows/unix incompatibility, i returned it

File: site/comps/BlogEntry.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					height: "30%",
+				},
+			}}/>
+
+			<View bg={"none"} dir="column">
+				<Text size="huge">{title}</Text>
+
+				<Text size="small" css={{
+					fontStyle: "italic",
+					color: "var(--color-fg3)",
```
Comment (slmjkdbtl) [inline site/comps/BlogEntry.tsx:51]: I think you can do `<Text size="small" italic color={3}>`

File: site/comps/BlogEntry.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	children,
+	title,
+	author,
+	date,
+	description,
+	image,
+	...args
+}) => {
+	return (
+		<View stretchX={true} height={180} dir="row" outlined={true} rounded={true} bg={3} padX={1} padY={1} css={{
```
Comment (slmjkdbtl) [inline site/comps/BlogEntry.tsx:27]: If a prop value is `true` you can omit it, like `<View stretchX rounded outlined></View>`

File: site/pages/blog.tsx [hunk 3]
```diff
@@ -1,13 +1,86 @@
...
+			title={blog.title}
+			author={blog.author}
+			date={blog.date}
+			description={blog.description}
+			image={`site/blog/banners/${blog.image}`}
+		/></a>)
+	}
+
+	return (
+		<>{blogsEntries}</>
```
Comment (slmjkdbtl) [inline site/pages/blog.tsx:38]: The more idiomatic way to do an array of components is
```js
return blogs.map((blog) => (
    <a><BlogEntry /></a>
))
```

**Review by slmjkdbtl — APPROVED** [review]
Looks good, thank you so much! Just some minor style feedbacks.

**Pull request merged**